activate.py

```diff
@@ -85,19 +108,22 @@ def main(eth1_endpoint, graph_endpoint, private_key, contract_address, contract_
     if len(verified) > 0:
         print("Doing activating keys transaction")
         for key in verified:
+            issuer = Issuer(a['abi'], contract_address, eth1_endpoint)
+            issuer.set_account(private_key)
             if Web3.fromWei(issuer.web3_eth.eth.get_balance(Web3.toChecksumAddress(contract_address)), "ether") > 32:
                 tx = issuer.depositToEth2(key)
                 issuer.do_transaction(tx)
             else:
                 print(
                     f"Balance of pool low for activation. \nCurrent balance: {Web3.fromWei(issuer.web3_eth.eth.get_balance(Web3.toChecksumAddress(contract_address)), 'ether')}")
+                del issuer
```

Comment:
Why specifically delete class object?

Reply:
reinitialising the object after every transaction, thats why deleting the object